packages/react/src/modules/modals/AddAttachmentToWorkspaceModal/AddAttachmentToWorkspaceModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { useTranslation } from 'react-i18next';
+import { Button, Modal } from '../../../components';
+import WorkspaceFoldersTree from './components/WorkspaceFoldersTree';
+import { useState } from 'react';
+
+interface AddAttachmentToWorkspaceModalProps {
+  /**
+   * Modal id (useful when multiple modal on the same page)
+   */
+  id?: string;
+  /**
+   * Is Modal Open
+   */
+  isOpen?: boolean;
+  /**
+   * Function to call when success button proceed
+   */
+  onSuccess?: () => void;
+  /**
+   * Function to call when closing modal
+   */
+  onCancel?: () => void;
+}
+
+export default function AddAttachmentToWorkspaceModal({
+  id = 'add-attachment-to-workspace-modal',
+  isOpen = false,
+  onSuccess = () => ({}),
+  onCancel = () => ({}),
+}: AddAttachmentToWorkspaceModalProps) {
+  const { t } = useTranslation();
+  const [selectedFolderId, setSelectedFolderId] = useState<string>('');
```

Comment:
J'aime bien autoriser la valeur `undefined` dans ce cas précis, je trouve cela plus clair à relire : 
```suggestion
  const [selectedFolderId, setSelectedFolderId] = useState<string|undefined>();
```
=> Typescript permet ensuite un meilleur contrôle des valeurs, `undefined` ou ID valide